easybuild/easyconfigs/l/LAST/LAST-1045-intel-2019b.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'ConfigureMake'
+
+name = 'LAST'
+version = '1045'
+
+homepage = 'http://last.cbrc.jp/'
+description = "LAST finds similar regions between sequences."
+
+toolchain = {'name': 'intel', 'version': '2019b'}
+toolchainopts = {'cstd': 'c++11'}
+
+source_urls = ['http://last.cbrc.jp/']
+sources = ['last-%(version)s.zip']
+checksums = ['6c4afb54594a0ec5a46603a61e381957a5e23daa0611e6872d2329264acd5c74']
+
+skipsteps = ['configure']
+
+buildopts = 'CC="$CC" CXX="$CXX" CFLAGS="$CFLAGS" CXXFLAGS="$CXXFLAGS -pthread -DHAS_CXX_THREADS"'
+installopts = 'prefix=%(installdir)s'
+
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in ['fastq-interleave',
+                                     'lastal',
+                                     'lastal8',
+                                     'lastdb',
+                                     'lastdb8',
+                                     'last-dotplot',
+                                     'last-map-probs',
+                                     'last-merge-batches',
+                                     'last-pair-probs',
+                                     'last-postmask',
+                                     'last-split',
+                                     'last-split8',
+                                     'last-train',
+                                     'maf-convert',
+                                     'maf-cut',
+                                     'maf-join',
+                                     'maf-sort',
+                                     'maf-swap',
+                                     'parallel-fasta',
+                                     'parallel-fastq']],
```

Comment:
Spreading this out vertically is a bit much, no? As long as they're sorted alphabetically, this is fine imho:

```python
sanity_check_paths = {
    'files': ['bin/%s' % x for x in ['fastq-interleave', 'lastal', 'lastal8', 'lastdb', 'lastdb8', 'last-dotplot',
                                     'last-map-probs', 'last-merge-batches', 'last-pair-probs', 'last-postmask',
                                     'last-split', 'last-split8', 'last-train', 'maf-convert', 'maf-cut',
                                     'maf-join', 'maf-sort', 'maf-swap', 'parallel-fasta', 'parallel-fastq']],
    'dirs': [],
}

Reply:
no big deal, fixed :+1: